src/wp-includes/l10n/class-wp-translation-file.php

```diff
@@ -0,0 +1,296 @@
+<?php
+/**
+ * I18N: WP_Translation_File class.
+ *
+ * @package WordPress
+ * @subpackage I18N
+ * @since 6.5.0
+ */
+
+/**
+ * Class WP_Translation_File.
+ *
+ * @since 6.5.0
+ */
+abstract class WP_Translation_File {
+	/**
+	 * List of headers.
+	 *
+	 * @since 6.5.0
+	 * @var array<string, string>
+	 */
+	protected $headers = array();
+
+	/**
+	 * Whether file has been parsed.
+	 *
+	 * @since 6.5.0
+	 * @var bool
+	 */
+	protected $parsed = false;
+
+	/**
+	 * Error information.
+	 *
+	 * @since 6.5.0
+	 * @var string|null Error message or null if no error.
+	 */
+	protected $error;
+
+	/**
+	 * File name.
+	 *
+	 * @since 6.5.0
+	 * @var string
+	 */
+	protected $file = '';
+
+	/**
+	 * Translation entries.
+	 *
+	 * @since 6.5.0
+	 * @var array<string, string>
```

Comment:
I think this is what was intended:

```suggestion
	 * @var array<string, string[]>
```